Gemfile

```diff
@@ -20,7 +20,6 @@ gem 'snappconfig'
 gem 'uglifier'
 
 source 'https://rails-assets.org' do
-  gem 'rails-assets-datetimepicker'
   gem 'rails-assets-moment'
```

Comment:
Do you feel like tackling one more?

Reply:
I feel like trying to tackle one more, sure. I got pretty myopic to a point.